opentechinstitute/luci-commotion
# Replace nodogsplash with fork of luci_splash

This pull request, in combination with https://github.com/opentechinstitute/commotion-feed/pull/68 and https://github.com/opentechinstitute/commotion-router/pull/167 replaces nodogsplash with a fork of luci_splash. It also adds commotion-dnsproxy to provide a splash screen even when an internet connection is not present. It addresses issues #294 and #137.

To test:
* Make sure that splash screen appears when connecting via the access point, whether an internet connection is present or not.
* Make sure that the splash screen never appears when connected through the ethernet interfaces.
* Test to see if other basic and advanced settings work as they did previously.
